compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformInsets.skiko.kt

```diff
@@ -72,3 +73,32 @@ internal fun PlatformInsets.exclude(insets: PlatformInsets) = PlatformInsets(
     right = (right - insets.right).coerceAtLeast(0.dp),
     bottom = (bottom - insets.bottom).coerceAtLeast(0.dp)
 )
+
+internal interface InsetsConfig {
+    val safeInsets: PlatformInsets
+        @Composable get
+
+    @Composable
+    fun excludeSafeInsets(content: @Composable () -> Unit)
+}
+
+internal object ZeroInsetsConfig : InsetsConfig {
```

Comment:
```suggestion
// TODO before make it public, think about how to merge `safeInsets` and `excludeSafeInsets` into one property or function
internal object ZeroInsetsConfig : InsetsConfig {
```

I lack context here, but from the first glance we have duplicated functionality here. This is not important, if it isn't public. But if it is public, the API should be minimal as possible.

Reply:
Disagree here. It's not duplicated because it might be provided not via local composition under the hood.